<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><meta name="description" content="PhyloTech advances human and environmental health by understanding life around us.  Our advanced molecular technology and bioinformatics algorithms simultaneously detect the presence and relative  abundance of more than 50,000 bacterial taxa in samples of any type"><meta name="keywords" content="Phylotech, human health, environmental health, technology, molecular technology, DNA, specimen, specimens, analysis, taxa, classifications, microbiome, microbiome analysis">
<title>Phylotech | Services | Analysis</title>
<LINK href="stylesheets/default.css" type=text/css rel=stylesheet>
<script type="text/javascript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
<style type="text/css">
<!--
.style7 {
	color: #00A981;
	font-weight: bold;
}
-->
</style>
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-17159665-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script></head>





<SCRIPT type="text/javascript" src="javascripts/milonic_src.js"></SCRIPT> 
<param copyright="JavaScript Menu by Milonic http://www.milonic.com"></param>
<script type="text/javascript">
<!-- 
if(ns4)_d.write("<SCR"+"IPT LANGUAGE=JavaScript SRC=javascripts/mmenuns4.js><\/SCR"+"IPT>"); 
else _d.write("<SCR"+"IPT LANGUAGE=JavaScript SRC=javascripts/mmenudom.js><\/SCR"+"IPT>"); 
-->
</SCRIPT>
<SCRIPT type="text/javascript" src="javascripts/menu_data.js"></SCRIPT> 
<body onload="MM_preloadImages('images/facebook2.gif','images/twitter2.gif')">

<table width="1000" border="0" align="center" cellpadding="0" cellspacing="0">
  <tr>
    <td><table width="1000" border="0" cellspacing="0" cellpadding="0">
      <tr>
        <td width="425"><a href="index.html"><img src="images/logo_sub.gif" alt="Phylotech - Back to Home" width="425" height="124" hspace="0" vspace="0" border="0" title="Phylotech - Back to Home" /></a></td>
        <td width="575"><table width="575" border="0" cellspacing="0" cellpadding="0">
          <tr>
            <td><img src="images/top_spacer.jpg" width="576" height="49" /><!--<table width="575" border="0" cellspacing="0" cellpadding="0">
              <tr>
                <td width="392"><img src="images/followus.gif" alt="Follow Us" title="Follow Us" width="392" height="49" /></td>
                <td width="79"><a href="#"><img src="images/facebook1.gif" alt="Click here to go to our Facebook Page..." title="Click here to go to our Facebook Page..." width="79" height="49" hspace="0" vspace="0" border="0" id="Image1" onmouseover="MM_swapImage('Image1','','images/facebook2.gif',1)" onmouseout="MM_swapImgRestore()" /></a></td>
                <td width="104"><a href="#"><img src="images/twitter1.gif" title="Click here to go to our Twitter Page..." alt="Click here to go to our Twitter Page..." width="105" height="49" hspace="0" vspace="0" border="0" id="Image2" onmouseover="MM_swapImage('Image2','','images/twitter2.gif',1)" onmouseout="MM_swapImgRestore()" /></a></td>
              </tr>
            </table>--></td>
          </tr>
          <tr>
            <td><SCRIPT TYPE="text/javascript">
               with(new menuname("Main Menu")){
               style=menuStyle;
               alwaysvisible=1;
               position="relative";
			   orientation="horizontal";

aI("image=images/nav_applications_off.jpg;overimage=images/nav_applications_on.jpg;showmenu=applications;url=applications_human_health.html");
aI("image=images/nav_services_on.jpg;overimage=images/nav_services_on.jpg;showmenu=services;url=services_works.html");
aI("image=images/nav_projects_off.jpg;overimage=images/nav_projects_on.jpg;showmenu=projects;url=projects_create.html");
aI("image=images/nav_about_off.jpg;overimage=images/nav_about_on.jpg;showmenu=about;url=about_company.html");
               }
               drawMenus();
            </SCRIPT></td>
          </tr>
          <tr>
            <td><img src="images/top_spacer.jpg" width="576" height="43" /></td>
          </tr>
        </table></td>
      </tr>
    </table></td>
  </tr>
  <tr>
    <td><img src="images/header_services_analysis.gif" alt="Services: Analysis" width="1001" height="63" /></td>
  </tr>
  <tr>
    <td></td>
  </tr>
  <tr>
    <td></td>
  </tr>
  <tr>
    <td background="images/bodybg.gif"><table border="0" cellspacing="0" cellpadding="0">
      <tr>
        <td width="135" align="left" valign="top"><IMG SRC="images/sidebg.jpg" width="135" height="17" /><br />
            <table width="135" border="0" cellpadding="0" cellspacing="0" background="images/sidebg.jpg">
              <tr background="images/sidebg.jpg">
                <td background="images/sidebg.jpg"><UL class=nav>
  <LI><IMG SRC="images/sidebullet_blank.gif" border="0" align="left" /><A href="services_works.html">How It Works</A>
  <LI><IMG SRC="images/sidebullet_blank.gif" border="0" align="left" /><A href="services_technology.html">Technology</A>
    <LI><IMG SRC="images/sidebullet2.jpg" border="0" align="left" />Analysis
	  <LI><IMG SRC="images/sidebullet_blank.gif"  border="0" align="left" /><A href="services_results.html">Results</A>
                  		  <LI><IMG SRC="images/sidebullet_blank.gif"  border="0" align="left" /><A href="services_faq.html">FAQ</A>
      	  <LI><IMG SRC="images/sidebullet_blank.gif"  border="0" align="left" /><A href="services_order.html">Order</A>
</UL></td>
              </tr>
            </table>
            <IMG SRC="images/sidebg.jpg" /><br />
<img src="images/leftfade.jpg" width="135" height="252" /></td>
        <td width="20" align="left" valign="top">&nbsp;</td>
        <td width="733" align="left" valign="top">
          <p class="header">Unique analysis exploits one-to-many relationships</p>
          <p class="bodyblue style7">Third-generation Technology</p>
          <p class="body">
PhyloChip assays and algorithms are in their third generation of development, having been optimized through focused efforts at LBNL.  Using over 1.1 million probes representing over 50,000 bacteria and archaea, our assay and algorithms are designed to leverage the one-to-many and many–to-many relationships that exist between probes and taxa.  Full-length 16S sequences are queried to power confidence in data interpretation. The rich representation of full-length 16S sequence has been demonstrated as a powerful approach for profiling complex bacterial populations in a wide range of environmental, animal, and human sample types and matrices. Reference sequences are built into each and every assay for quality management and bench marking quantification. 
</p>
          <p class="bodyblue style7">Sensitivity and Specificity</p>
          <p class="body"> We’ve become aware that it is challenging to assay all known bacterial species by pyrosequencing.  Recently published reports of artifacts and anomalies with these methods describe problems linked to data quality assumptions.  A combination of an unacceptable high frequency of errors in base calling and artifacts in taxonomic placement leads to inaccurate and non-reproducible descriptions of community composition.  Validation studies have revealed irregularities in determination of relative abundance.  Furthermore, there is a concern over the incidence of human subject DNA contaminating sequence reads and inadvertent publication of personal DNA, violating protection of patient privacy.  Moreover, sufficiently deep sequencing, to enable detection of lower abundance bacteria and archaea,  appears to limit the number of samples that can be analyzed, thereby dramatically reducing the power of statistical analyses for discernment of microbial patterns under various human and environmental conditions.</p>
          <p class="body"><a href="contact_applications.html"><br />
          Contact us to learn more. </a></p></td>
        </tr>
    </table></td>
  </tr>
  <tr>
    <td><img src="images/bottomcurves.gif" width="1001" height="35" /></td>
  </tr>
  <tr>
    <td height="35"><table width="1000" border="0" cellspacing="0" cellpadding="0">
      <tr>
        <td width="14">&nbsp;</td>
        <td width="486" align="left" class="style5"><a href="about_privacy.html" class="foot">Privacy Policy</a> &nbsp;  | &nbsp; <a href="about_terms.html" class="foot">Terms of Use</a> &nbsp; | &nbsp; <a href="about_contact.html" class="foot">Contact Us</a></td>
        <td width="489" align="right" class="body">  © 2010 PhyloTech, Inc.</td>
        <td width="11">&nbsp;</td>
      </tr>
    </table></td>
  </tr>
</table>


</body>
</html>
