<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><meta name="description" content="PhyloTech advances human and environmental health by understanding life around us.  Our advanced molecular technology and bioinformatics algorithms simultaneously detect the presence and relative  abundance of more than 50,000 bacterial taxa in samples of any type"><meta name="keywords" content="Phylotech, human health, environmental health, technology, molecular technology, DNA, specimen, specimens, analysis, taxa, classifications, microbiome, microbiome analysis">
<title>Phylotech | Services | Frequently Asked Questions</title>
<LINK href="stylesheets/default.css" type=text/css rel=stylesheet>
<script type="text/javascript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-17159665-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>





<SCRIPT type="text/javascript" src="javascripts/milonic_src.js"></SCRIPT> 
<param copyright="JavaScript Menu by Milonic http://www.milonic.com"></param>
<script type="text/javascript">
<!-- 
if(ns4)_d.write("<SCR"+"IPT LANGUAGE=JavaScript SRC=javascripts/mmenuns4.js><\/SCR"+"IPT>"); 
else _d.write("<SCR"+"IPT LANGUAGE=JavaScript SRC=javascripts/mmenudom.js><\/SCR"+"IPT>"); 
-->
</SCRIPT>
<SCRIPT type="text/javascript" src="javascripts/menu_data.js"></SCRIPT> 
<body onload="MM_preloadImages('images/facebook2.gif','images/twitter2.gif')">

<table width="1000" border="0" align="center" cellpadding="0" cellspacing="0">
  <tr>
    <td><table width="1000" border="0" cellspacing="0" cellpadding="0">
      <tr>
        <td width="425"><a href="index.html"><img src="images/logo_sub.gif" alt="Phylotech - Back to Home" width="425" height="124" hspace="0" vspace="0" border="0" title="Phylotech - Back to Home" /></a></td>
        <td width="575"><table width="575" border="0" cellspacing="0" cellpadding="0">
          <tr>
            <td><img src="images/top_spacer.jpg" width="576" height="49" /><!--<table width="575" border="0" cellspacing="0" cellpadding="0">
              <tr>
                <td width="392"><img src="images/followus.gif" alt="Follow Us" title="Follow Us" width="392" height="49" /></td>
                <td width="79"><a href="#"><img src="images/facebook1.gif" alt="Click here to go to our Facebook Page..." title="Click here to go to our Facebook Page..." width="79" height="49" hspace="0" vspace="0" border="0" id="Image1" onmouseover="MM_swapImage('Image1','','images/facebook2.gif',1)" onmouseout="MM_swapImgRestore()" /></a></td>
                <td width="104"><a href="#"><img src="images/twitter1.gif" title="Click here to go to our Twitter Page..." alt="Click here to go to our Twitter Page..." width="105" height="49" hspace="0" vspace="0" border="0" id="Image2" onmouseover="MM_swapImage('Image2','','images/twitter2.gif',1)" onmouseout="MM_swapImgRestore()" /></a></td>
              </tr>
            </table>--></td>
          </tr>
          <tr>
            <td><SCRIPT TYPE="text/javascript">
               with(new menuname("Main Menu")){
               style=menuStyle;
               alwaysvisible=1;
               position="relative";
			   orientation="horizontal";

aI("image=images/nav_applications_off.jpg;overimage=images/nav_applications_on.jpg;showmenu=applications;url=applications_human_health.html");
aI("image=images/nav_services_on.jpg;overimage=images/nav_services_on.jpg;showmenu=services;url=services_works.html");
aI("image=images/nav_projects_off.jpg;overimage=images/nav_projects_on.jpg;showmenu=projects;url=projects_create.html");
aI("image=images/nav_about_off.jpg;overimage=images/nav_about_on.jpg;showmenu=about;url=about_company.html");
               }
               drawMenus();
            </SCRIPT></td>
          </tr>
          <tr>
            <td><img src="images/top_spacer.jpg" width="576" height="43" /></td>
          </tr>
        </table></td>
      </tr>
    </table></td>
  </tr>
  <tr>
    <td><img src="images/header_services_faq.jpg" alt="Services: Frequently Asked Questions" width="1001" height="63" /></td>
  </tr>
  <tr>
    <td></td>
  </tr>
  <tr>
    <td></td>
  </tr>
  <tr>
    <td background="images/bodybg.gif"><table border="0" cellspacing="0" cellpadding="0">
      <tr>
        <td width="135" align="left" valign="top"><IMG SRC="images/sidebg.jpg" width="135" height="17" /><br />
            <table width="135" border="0" cellpadding="0" cellspacing="0" background="images/sidebg.jpg">
              <tr background="images/sidebg.jpg">
                <td background="images/sidebg.jpg"><UL class=nav>
  <LI><IMG SRC="images/sidebullet_blank.gif" border="0" align="left" /><A href="services_works.html">How It Works</A>
  <LI><IMG SRC="images/sidebullet_blank.gif" border="0" align="left" /><A href="services_technology.html">Technology</A>
    <LI><IMG SRC="images/sidebullet_blank.gif" border="0" align="left" /><a href="services_analysis.html">Analysis</A>
	  <LI><IMG SRC="images/sidebullet_blank.gif"  border="0" align="left" /><A href="services_results.html">Results</A>
            	  <LI><IMG SRC="images/sidebullet2.jpg"  border="0" align="left" />FAQ
      		  <LI><IMG SRC="images/sidebullet_blank.gif"  border="0" align="left" /><A href="services_order.html">Order</A>
</UL></td>
              </tr>
            </table>
            <IMG SRC="images/sidebg.jpg" /><br />
<img src="images/leftfade.jpg" width="135" height="152" /></td>
        <td width="20" align="left" valign="top">&nbsp;</td>
        <td width="733" align="left" valign="top">
          <p class="header">You Have Questions. We Have Answers.</p>
          <p class="bodyblue"><strong>How much DNA should I send?           </strong></p>
          <!--   <ul class="body">
            <li class="body">$970 per sample for extracted DNA              </li>
            <li class="body">15 sample minimum initial order quantity per project              </li>
            <li class="body">5 sample minimum follow-on order quantity per project              </li>
            <li class="body">$25 surcharge for each DNA sample isolated  from  frozen water filters              </li>
            <li class="body">10% discount on single purchase orders over 100 samples              </li>
            <li class="body">Additional discounts for standing orders > 100 samples/month </li>
          </ul>-->
          <p class="body">
If possible send an excess, 0.5 – 1 microgram, of each specimen. Amplification of bacteria and archaea are carried out separately, using 10-30 ng per reaction, totaling 20 – 60 ng. If the amplification yield is low, a second amplification will be carried out and the amplicons pooled (2 x 20-60 ng = 40-120 ng).  Taking into account quantification variation due to differences in DNA measurement instrumentation, sublimation in deep storage, etc., it is best to send an excess if possible.<br />
          <br />
          </p>
          <p class="bodyblue"><strong>How concentrated should the isolated DNA be when I ship it?            </strong></p>
          <p class="body">10 – 30 ng/μl.            <br />
            <br />
          </p>
          <p class="bodyblue"><strong>How much DNA is in a bacterial cell?            </strong></p>
          <p class="body">4.2 fg/cell   (Kubitschek, 1971) <br />
            <br />
          </p>
          <p class="bodyblue"><strong>Who gets my data?            </strong></p>
          <p class="body">Your data is your data.  You will receive a secure weblink to your data, figures and tables. Your data will be accessible for 2 years after your project is completed. After that, it is archived but still accessible. <br />  
              <br />
          </p>
          <p class="bodyblue"><strong>I have an Affymetrix machine. Can I buy the chip directly?            </strong></p>
          <p class="body">Right now, we're processing everything in a standard format to ensure high quality results.  
            In the future, we envision standardizing protocols and making the chips available to run in your own lab. However, we'll continue to provide data analysis services. <br />  
            <br />
          </p>
          <p class="bodyblue"><strong>Why can't I do the data analysis myself?            </strong></p>
          <p class="body">We’re geared towards customers who want to answer biological and ecological questions. We offer basic analysis, providing a full data download, including tables and figures. You can analyze your basic data however you wish. </p>
          <p class="body">Regarding the raw data analysis, the PhyloChip assay is unique in the way that the probes are selected and the way they identify bacterial species. We selected the most specific probes for each operational taxonomic unit, OTU. One probe can fit into many OTUs because there are so many similar 16s rRNA sequence strings or cassettes. One probe can be informative for many taxa.  In addition, there are multiple probes per OTU. Our data analysis algorithm enables us to discern which OTU is being detected by configurations of multiple probes and weigh that accordingly in determining presence and relative abundance. Leveraging many years of statistical and bioinformatics expertise, we conduct these analysis steps for you.            <br />
            <br />
          </p>
          <p class="bodyblue"><strong>Isn’t next generation sequencing more sensitive? Faster? Better for discovery?            </strong></p>
          <p class="body">PhyloChip technology is the most comprehensive microbial assay available today with over 1.1 million probes, representing over 50,000 bacterial and archaeal taxa. A key feature setting PhyloChip technology apart from other methods is the use of multiple oligonucleotide probes for every known category of prokaryotic organism, allowing highly sensitive and high-confidence detection (DeSantis et al., 2005). It has been demonstrated that PhyloChip technology results in several-fold increase in detected microbial diversity over the clone library method. One of the reasons for this is the high sensitivity of the method, with the ability to detect organisms present at a fraction below 10-4 abundance of the total sample (DeSantis et al., 2007). Experiments using sequence-specific PCR have confirmed that taxa identified by the PhyloChip assay, but not clone libraries, were indeed present in the original environmental samples (DeSantis et al., 2007, La Duc et al., 2009). A strong linear correlation has been confirmed between microarray probe set intensity and concentration of OTU specific 16S rRNA gene copy number, allowing quantification across a wide dynamic range. Validation experiments have demonstrated that intensity-based quantification is highly repeatable with coefficient of variation (CV) between replicate assays of 5-10%, with most CVs less than 7%. The PhyloChip assay has been successfully used on a variety of complex sample types and the resulting findings have been confirmed by additional methods, including qPCR and 16S rRNA gene clone libraries (Brodie et. al 2006, Brodie et al. 2007, DeSantis et al. 2007, La Duc et al. 2009).  Studies at the Lawrence Berkley National Laboratories by Andersen et al., using split-samples have confirmed that >90% of all 16S rRNA sequence types identified by the more expensive clone library method are also identified by the PhyloChip assay. Although each sample assessed using the PhyloChip assay and analysis provides detailed information on microbial composition, the highly parallel and reproducible nature of the assay allows tracking community dynamics over time and treatment. With no prior knowledge, specific microbial interactions may be identified that are key to particular changing environments. <br />
              <br />
            Recently, reports of artifacts and anomalies with pyrosequencing have been published (Quince  et al., 2009, Kunin et al., 2010).  Many of the problems reported can be traced to data quality assumptions. Pyrosequencing methods result in a higher error rate than Sanger sequencing (Aury et al., 2008).  This limitation is overcome in shotgun sequencing by aligning multiple reads per loci and finding the predominant base call for each position.  However, since there is no assembly step for microbial population samplings, each read must be interpreted on its own.  Therefore, the combination of high frequency of errors in base calling resulting from next generation sequencing methods and using only a single observation per taxonomic assignment leads to an inaccurate and non-reproducible description of community composition.</p>
          <p class="body"><em> To enable comparable results you would have to spend 4-6 times as much on pyrosequencing and many months assembling and interpreting results. </em><br />
            <br />
          </p>
          <p class="bodyblue"><strong>Why do I need a minimum of 15 samples to initiate a project?            </strong></p>
          <p class="body">The minimum project size of 15 samples adequately powers the analysis and ensures robust statistics and meaningful data reports. <br />  
              <br />
          </p>
          <p class="bodyblue"><strong>Are there publications about PhyloChip technology?          </strong></p>
          <p class="body">There are many publications using the PhyloChip™ G2, second generation, technology. Work using the third generation PhyloChip is just now being submitted (June 2010). The third generation assay is the result of development work on probe selection, the assay and analytical algorithms and has more probes, representing more bacteria, and more control sequences for more comprehensive coverage, improved sensitivity and enhanced built-in quantification and quality control. </p>
          <p class="body"><hr /></p>
          <p class="body"><em><strong>References  </strong></em><br /><br />


Aury J.M. <em>et al.</em>, (2008)  High quality draft sequences for prokaryotic genomes using a mix of new sequencing technologies. <em>BMC Genomics</em> 2008, 9:603.
<br /><br />
Brodie, E. L. <em>et al</em>, (2006) Application of a high-density oligonucleotide microarray approach to study bacterial population dynamics during uranium reduction and reoxidation. <em>Appl. Env. Micro</em>. 72:6288-6298.
<br /><br />
Brodie, E. L. <em>et al.</em>, (2007) Urban aerosols harbor diverse and dynamic bacterial populations. <em>Proc Natl Acad Sci. </em>104:299-304.
<br /><br />
DeSantis, T. Z. <em>et al</em>., (2005) Rapid quantification and taxonomic classification of environmental DNA from both prokaryotic and eukaryotic origins using a microarray. <em>FEMS Micro. Lett. </em>245: 271-278.
<br /><br />
DeSantis, T.Z. <em>et al</em>., (2007) High-density universal 16S rRNA microarray analysis reveals broader diversity than typical clone library when sampling the environment. <em>Microb Ecol</em>, 53(3): p. 371-83.
<br /><br />
Kubitschek, HE and Friedman, MC (1971) Chromosome replication and the division cycle of Escherichia coli B/r. Journal of Bacteriology. 107:95-99.
<br /><br />
Kunin V. <em>et al.</em>, (2010) Wrinkles in the rare biosphere: pyrosequencing errors can lead to artificial inflation of diversity estimates. Environ Microbiol 2010, 12:118-123.
La Duc M.T. et al., (2009) Comprehensive census of bacteria in clean rooms by using DNA microarray and cloning methods.  <em>Appl Environ Microbiol.</em> Oct;75(20):6559-67.
<br /><br />
Quince C,<em> et al.</em>, (2009) : Accurate determination of microbial diversity from 454 pyrosequencing data. <em>Nat Methods</em> 2009, 6:639-641</p>
          <p class="body">&nbsp;</p></td>
        </tr>
    </table></td>
  </tr>
  <tr>
    <td><img src="images/bottomcurves.gif" width="1001" height="35" /></td>
  </tr>
  <tr>
    <td height="35"><table width="1000" border="0" cellspacing="0" cellpadding="0">
      <tr>
        <td width="14">&nbsp;</td>
        <td width="486" align="left" class="style5"><a href="about_privacy.html" class="foot">Privacy Policy</a> &nbsp;  | &nbsp; <a href="about_terms.html" class="foot">Terms of Use</a> &nbsp; | &nbsp; <a href="about_contact.html" class="foot">Contact Us</a></td>
        <td width="489" align="right" class="body">  © 2010 PhyloTech, Inc.</td>
        <td width="11">&nbsp;</td>
      </tr>
    </table></td>
  </tr>
</table>


</body>
</html>
